src/common-principles.md

```diff
@@ -238,10 +238,14 @@ distinguish partial results from the raw data and share the latter.
 See [Storage of derived datasets](#storage-of-derived-datasets) for more on
 organizing derivatives.
 
-Similar rules apply to source data, which is defined as data before
-harmonization, reconstruction, and/or file format conversion (for example, E-Prime event logs or DICOM files).
-Storing actual source files with the data is preferred over links to
-external source repositories to maximize long term preservation,
+Similar rules apply to source data, which is defined as data
+before harmonization, reconstruction, and/or file format conversion
+(for example, E-Prime event logs or DICOM files).
+Retaining the source data is especially valuable
+in a case when conversion elicits the loss of crucial metadata
```

Comment:
This use of "elicit" feels unusual.

```suggestion
in a case when conversion fails to preserve crucial metadata
```

Reply:
true... in my (poor) defense this was just extracted from one the modality specific page... 😜